spmd/compiler/fusion.py

```diff
@@ -93,6 +89,8 @@ class GraphInfo:
     actual_grad_index_mapping: Dict[fx.Node, int] = field(
         default_factory=lambda: {}
     )
+    global logger
+    logger = get_logger("graph_opt")  # type: ignore
```

Comment:
I think we should probably move this out of the GraphInfo data class? logging probably should not correlate with custom data structure

Reply:
This is the part that I don't like about this PR but for now it allows us to configure what logging we want. Answered above as to why initializing a global python object won't work.